### PR TITLE
Added Counterspell Casablanca!

### PR DESCRIPTION
**Pull request by slippyishappy**
The Counterspell Casablanca Hackathon Website.

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/casablanca**](https://counterspell-em14esi6b.hackclub.dev/casablanca) |
|:-:|
|<a href="https://counterspell-em14esi6b.hackclub.dev/casablanca">Screenshot of /casablanca</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/AW1-myPh8HgOFSAyBMsuJIXOE8qew_Z_J8Ltyd9hBNSqbbcclLOnNYt2r6DuyQM7vUUKnarKQBFpyHr8aA9PQA.png">(view full size)</a> |







Commit 5665a7255792e5657a347191253b39c63ad8e8b7 (https://counterspell-em14esi6b.hackclub.dev).

File: yarn.lock
Comment (samdev-7) [inline yarn.lock:1]: The packages updated recently, you might need to re-run yarn install or revert this change.

**Review by samdev-7 — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (slippyishappy) [reply yarn.lock:1]: I've just updated the packages as you suggested. Let me know if there's anything else I need to do!

Comment (samdev-7) [reply yarn.lock:1]: Not sure what's happening with this file. Can you revert/restore this to what's on the main repo?

Comment (slippyishappy) [reply yarn.lock:1]: I fetched the latest changes from this repo and rebased my branch. Is the file good to go now?

**Review by samdev-7 — CHANGES_REQUESTED** [review]
Not sure what happened here, but it seems like you’re trying to edit all previous cities?

**Comment by slippyishappy** [comment]
I basically tried to fetch everything in this repo to mine, but since there are many issues, I can close this pr, fork this repo again, make my changes, and make a new pr.

**Pull request closed without merging**